_posts/2024-10-28-boosting-cde-security-on-k8s-cluster.adoc

```diff
@@ -0,0 +1,80 @@
+---
+title: "Boosting CDE Security on k8s cluster: A Guide to new features"
+layout: post
+author: Anatolii Bazko
+description: >-
+  Boosting CDE Security on k8s cluster: A Guide to new features
+categories: []
+keywords: ['workspace', 'security', 'administrator']
+slug: /@tolusha/boosting-cde-security-on-k8s-cluster
+---
+
+== Introduction
+
+As organizations transition to cloud-native development, maintaining control, security, and flexibility in development environments becomes critical. Eclipse Che is a powerful Kubernetes-native IDE that offers administrators tools for ensuring security and resource efficiency. In this post, we'll look at three new Eclipse Che features that improve security and resource control in your development environments.
+
+== Limiting the Number of Simultaneous Workspaces
+
+Effective resource management is critical in cloud-based environments, as uncontrolled workspace development can quickly deplete resources and degrade system performance. To mitigate this risk, Eclipse Che now has a functionality that restricts the amount of running workspaces per cluster.
+
+Administrators can use this feature to limit the amount of workspaces that can run at the same time, reducing resource misuse, supports fair allocation of resources among teams, hence increasing overall efficiency.
+
+In the example below, we set the maximum number of running workspaces per cluster to 20:
+
+[source,shell]
+----
+kubectl patch checluster/eclipse-che \
+  --namespace eclipse-che \
+  --type='merge' -p \
+    '{"spec":
+        {"devEnvironments":
+          {"maxNumberOfRunningWorkspacesPerCluster": 20}
+        }
+     }'
+----
+
+If a user exceeds the workspace limit, they will receive a warning message during workspace startup: "Exceeded the cluster limit for running DevWorkspaces"
+
+image::/assets/img/boosting-cde-security-on-k8s-cluster/exceeded-the-cluster-limit-for-running-devworkspaces.png[Exceeded the cluster limit for running DevWorkspaces]
+{nbsp} +
+
+To learn more about configuring workspace limits, follow the link:https://eclipse.dev/che/docs/stable/administration-guide/limiting-the-number-of-workspaces-that-all-users-can-run-simultaneously[Eclipse Che documentation].
+
+== Configuring Allowed URLs
```

Comment:
@tolusha Before the "Allowed URLs" section I would add "Untrusted Repository Warning" section which would explain first https://github.com/eclipse-che/che/issues/23018